lib/oembed_proxy/npr.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'oembed_proxy/inactive_support'
+
+module OembedProxy
+  # NPR Fauxembed
+  class Npr
+    using InactiveSupport
+    NPR_REGEX = %r{https:\/\/.*\.npr\.org\/*}.freeze
```

Comment:
This pattern anchors the match to the start of the string, has stricter hostname matching, and does what I think was intended at the end of the pattern (matching one or more characters after the `.org/`):

```suggestion
    NPR_REGEX = %r{\Ahttps:\/\/(?:[a-z0-9-]+\.)+npr\.org\/.+}.freeze
```